content/blog/2021/06/introducing-the-psp-host-namespaces-policy.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Introducing the PSP host namespaces policy
+authors:
+    - Rafael Fernández López
+date: 2021-06-11
+---
+
+As you probably know, Kubernetes Pod Security Policies [are being
+deprecated in Kubernetes
+1.21](https://github.com/kubernetes/kubernetes/blob/a3abd06ad53b2f02dcb6e060c7606ceda41f44af/CHANGELOG/CHANGELOG-1.21.md#deprecation-of-podsecuritypolicy)
+-- although these APIs will be served until Kubernetes 1.25 it's a
+good time to start thinking about what you will use to replace them.
+
+At Kubewarden we have an ongoing effort to replace the Pod Security
+Policies with small, targeted Kubewarden policies.
+
+Up until now, we have implemented some policies that replace some Pod
```

Comment:
where are these PSPs that are being replaced ? Can you add a link here ?

Reply:
Yes, they are listed on the next lines :)